Fabric.Authorization.Persistence.SqlServer/EntityModels/Client.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Fabric.Authorization.Domain.Models;
+
+namespace Fabric.Authorization.Persistence.SqlServer.EntityModels
+{
+    public class Client : ITrackable, ISoftDelete
+    {
+        public int Id { get; set; }
+        public string ClientId { get; set; }
+        public string Name { get; set; }
```

Comment:
should this be nullable, or is it required when creating a client?

Reply:
we validate it as a required field when posting a new one

Reply:
good catch, it should be nullable. i will update that